src/components/features/Login/Login.jsx

```diff
@@ -0,0 +1,194 @@
+// import { useEffect } from 'react';
+
+import { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+
+import { REGEXFOREMAIL as regexEmail } from '../../../config/config.jsx';
+import useInput from '../../../hooks/useInput.jsx';
+import { useAuthContext } from '../../../stores/AuthContext.jsx';
+// eslint-disable-next-line no-unused-vars
+import AsideMenu from '../../Layout/AsideMenu//AsideMenu.jsx';
+// import TopBar from '../../Layout/Topbar/TopBar.jsx';
+
+// Import styles:
+import './Login.scss';
+// import LoginForm from './LoginForm.jsx';
+
+// Helpers:
+const isEmail = (email) => regexEmail.test(email);
+const isPassword = (password) =>
+  password.trim() !== '' && password.length >= 6 && password.length <= 20;
+
+const Login = () => {
+  const { user, login } = useAuthContext();
+  // eslint-disable-next-line no-unused-vars
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+
+  const {
+    value: enteredEmail,
+    isValid: enteredEmailIsValid,
+    hasError: emailInputHasError,
+    handleInputChange: handleEmailInputChange,
+    handleInputBlur: handleEmailInputBlur,
+    reset: resetEmailInput,
+  } = useInput(isEmail);
+
+  const {
+    value: enteredPassword,
+    isValid: enteredPasswordIsValid,
+    hasError: passwordInputHasError,
+    handleInputChange: handlePasswordInputChange,
+    handleInputBlur: handlePasswordInputBlur,
+    reset: resetPasswordInput,
+  } = useInput(isPassword);
+
+  let formIsValid = false;
+
+  // Check for submit button:
+  if (enteredEmailIsValid && enteredPasswordIsValid) {
+    formIsValid = true;
+  }
+
+  // Check and set classes:
+  const emailInputClasses = emailInputHasError
```

Comment:
Use a package like https://www.npmjs.com/package/clsx for easier and more readable css class logic :)